deno/payloads/v10/user.ts

```diff
@@ -145,6 +145,10 @@ export enum UserFlags {
 	 * User has been identified as spammer
 	 */
 	Spammer = 1 << 20,
+	/**
+	 * @unstable User's account has been quarantined based on recent activity
+	 */
```

Comment:
```suggestion
	/**
	 * User's account has been quarantined based on recent activity
	 *
	 * @unstable This User flag is currently not documented by Discord but has known values which we will try to keep up to date.
	 */
```
Probably should also apply to Spammer, as said in https://github.com/discordjs/discord-api-types/pull/495#discussion_r900419075.